homeassistant/components/tibber/sensor.py

```diff
@@ -213,7 +213,7 @@
         key="month_cost",
         name="Monthly cost",
         device_class=SensorDeviceClass.MONETARY,
-        state_class=SensorStateClass.MEASUREMENT,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
This is the cost per month and it resets every month. We need to set `last_reset` otherwise the calculation will be incorrect.

https://developers.home-assistant.io/docs/core/entity/sensor#entities-representing-a-total-amount

See https://github.com/home-assistant/core/pull/82187

Reply:
Okay, I will leave this for now, until there is decided on the correct method.
Note: current setup gives errors in the logs for incompatibility between the type of state_class and device_class

Reply:
Please open a PR and remove the state class for this sensor kind for now until we can implement it correctly.

Reply:
https://github.com/home-assistant/core/pull/88542